src/components/transactions/TxDetails/Summary/TxDataRow/styles.module.css

```diff
@@ -26,6 +26,11 @@
   align-items: center;
 }
 
+/* 2nd only exists if there is address book entry */
+.address > div > div > div:nth-child(2) {
+  margin-top: -4px;
+}
```

Comment:
This works but we need to keep in mind that it highly depends on the structure of EthHashInfo. As an alternative we could also decrease the line-height of the first child although its not as effective but assumes less of the structure e.g.

```
.address > div {
  line-height: 1;
}
```

@liliiaorlenko did we consider also increasing the spacing between each row like this (4px)?
<img width="657" alt="Screenshot 2023-07-06 at 15 43 00" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/2a6a2adf-89d7-462e-99af-84c698be93d2">